controllers/channel.js

```diff
@@ -413,11 +413,32 @@ exports.closeChannel = (req,res) => {
 exports.listForwards = (req,res) => {
     function connFailed(err) { throw err }
     ln.on('error', connFailed);
-
+    var {offset, maxLen, reverse} = req.params
```

Comment:
this should be:
`var {offset, maxLen, reverse} = req.query`

Reply:
used req.params, cause I was parsing it via params in URL, wanted to keep it homogeneous with rest of the code 😅  , you sure you want me to remove the optional params in routes and change it to query ?

Reply:
The standard which we are following (for this library) is to pass all the optional parameters for GET calls as query params.